LenovoLegionToolkit.Cmdline/Flags.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace LenovoLegionToolkit.Cmdline;
+
+public class Flags
+{
+    public bool ShowHelp { get; }
+    public bool Silent { get; }
+    public string? QuickActionRunName { get; }
+
+    public bool Error { get; }
+
+    public Flags(IEnumerable<string> startupArgs)
+    {
+        var args = startupArgs.ToArray();
+        try
+        {
+            ShowHelp = BoolValue(args, "--help");
+            Silent = BoolValue(args, "--silent");
+            QuickActionRunName = StringValue(args, "--run");
```

Comment:
In case the CLI interface is extended, this parameter name is a bit too generic. Something like `--quickAction` would be better.